client/src/components/views/Orders.jsx

```diff
@@ -0,0 +1,236 @@
+import React, { Fragment, useState, useRef } from 'react';
+import {
+  Center,
+  useToast,
+  Textarea,
+  Select,
+  useDisclosure,
+  Table,
+  Thead,
+  Tbody,
+  Tr,
+  TableCaption,
+  Heading,
+  Button,
+  FormControl,
+  FormLabel,
+  Input,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+} from '@chakra-ui/react';
+import { SmallAddIcon } from '@chakra-ui/icons';
+import Loader from 'components/common/Loader.jsx';
+import { getOrders, getMaterialList, postOrders } from 'utils/api/orders.js';
+import { useQuery } from 'react-query';
+import { StyledTableRow, StyledTableHeader, StyledTableCell } from 'components/common/Table.jsx';
+import { TablePagination } from '@material-ui/core';
+import { NoResultImage } from 'components/common/Image.jsx';
+
+const Orders = () => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const { isLoading, isSuccess, data, refetch } = useQuery('orders', getOrders);
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(10);
+  const toast = useToast();
+  const [material, setMaterial] = useState();
+  const [cost, setCost] = useState(0);
+  const [deliveryDate, setDeliveryDate] = useState();
+  const [orderDate, setOrderDate] = useState();
+  const [quantity, setQuantity] = useState(0);
+  const manufacturer = useRef('');
+  const location = useRef('');
+  const note = useRef('');
+  const materialTypes = ['rubber', 'aluminum', 'steel', 'copper', 'plastic', 'leather'];
+  const materialCost = useQuery('orders/materialList', getMaterialList);
```

Comment:
I think you can get the list of materialTypes from the `orders/materialList` endpoint already, it'd be more consistent

Reply:
looking into it